src/constants.ts

```diff
@@ -11,3 +11,17 @@ export const productionEnvironmentName: string = 'Production';
 export const appSubpathSetting: string = 'appSubpath';
 export const apiSubpathSetting: string = 'apiSubpath';
 export const appArtifactSubpathSetting: string = 'appArtifactSubpath';
+
+// Doc for these parameter values: https://developer.github.com/v3/checks/runs/#parameters
+
+export enum Conclusion {
```

Comment:
These types should stay in the same file as the type for `GitHubAction` since they're so closely related. If you don't like having it in `ActionTreeItem`, make a separate file for GitHub stuff. The separate file would somewhat represent a drop-in replacement for the GitHub npm package that I assume you guys will switch to soon

Reply:
I think you missed this comment-- we probably could move this to the gitHubTypings.  Annoying that they don't have their own enum for this, but alas

Reply:
Oops! Will do.